jjb/dynamic/scale-ci_scale.yml

```diff
@@ -4,51 +4,24 @@
     builders:
     - shell: |+
         set -o pipefail
-        set -eux
+        set -ex
 
         # get perf keys to access orchestration host and set ssh session options
         git clone https://${SSHKEY_TOKEN}@github.com/redhat-performance/perf-dept.git
         export PUBLIC_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_pbench_ec2.pub
         export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_pbench_ec2
-        export OPTIONS="-o StrictHostKeyChecking=no -o ServerAliveInterval=30 -o ServerAliveCountMax=1200"
+        export OPTIONS="-o StrictHostKeyChecking=no -o ServerAliveInterval=1 -o ConnectionAttempts=100"
         chmod 600 ${PRIVATE_KEY}
 
         # fetch the kubeconfig from the orchestration host
         echo "Fetching the kubeconfig from the orchestration host"
-        scp ${OPTIONS} -i ${PRIVATE_KEY} ${ORCHESTRATION_USER}@${ORCHESTRATION_HOST}:${KUBECONFIG_FILE} ${WORKSPACE}/kubeconfig
+        scp ${OPTIONS} -i ${PRIVATE_KEY} ${ORCHESTRATION_USER}@${ORCHESTRATION_HOST}:$HOME/.kube/config ${WORKSPACE}/kubeconfig
```

Comment:
We need to provide a way to specify the kubeconfig location to copy it from since it might not be in the home dir in some of the cases. For example, shift on stack clusters are going to have the kubeconfig under a different location.